spa/src/appStore.ts

```diff
@@ -77,6 +78,12 @@ const appStore = Vue.observable<AppStore>({
     setPlace(placeData: Place): void{
       appStore.data.place = placeData;
     },
+    getGestures(): [] {
```

Comment:
I would opt for keeping the gestures as a `string[]` on the User interface and parsing them on the way in to the store, as opposed to on the way out. This could (should?) be done in a `setUser` function on the store.